src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(number) {
+    if (!arguments.length) {
+      const oldSum = sum;
+
+      sum = 0;
+
+      return oldSum;
+    }
+
+    sum += +number;
```

Comment:
I'd use `Number()` constructor, `+` is too tiny to notice
```suggestion
    sum += Number(number);
```